app/graphql/generated.ts

```diff
@@ -61,12 +61,14 @@ export type Scalars = {
 }
 
 export type Account = {
+  readonly btcWallet?: Maybe<Wallet>
```

Comment:
isn't `?` redundant when using `Maybe` type?

Reply:
Don't know for sure. But it's a generated files